### ISSUE TITLE
installing sfdx-cli@7.30.11 using npm fails with `No matching version found for salesforcedx@47.3.4.`

### ISSUE THREAD

**Issue by amtrack** [forcedotcom/cli#211]
### Summary

Installing (or updating to) sfdx-cli@7.30.11 using npm fails with `No matching version found for salesforcedx@47.3.4.`

### Steps To Reproduce:

1.  npm install --global sfdx-cli

### Expected result

Installation / update should work.

### Actual result

```
Command failed: npm install --global sfdx-cli@7.30.11 --color=always
npm ERR! code ETARGET
npm ERR! notarget No matching version found for salesforcedx@47.3.4.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget 
npm ERR! notarget It was specified as a dependency of 'sfdx-cli'
npm ERR! notarget
```

### Additional information

**SFDX CLI Version**(to find the version of the CLI engine run sfdx --version):
sfdx-cli/7.28.7 darwin-x64 node-v12.12.0

**SFDX plugin Version**(to find the version of the CLI plugin run sfdx plugins --core)
@oclif/plugin-commands 1.2.3 (core)
@oclif/plugin-help 2.2.1 (core)
@oclif/plugin-not-found 1.2.3 (core)
@oclif/plugin-plugins 1.7.8 (core)
@oclif/plugin-update 1.3.9 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/sfdx-trust 3.0.5 (core)
analytics 1.2.1 (core)
generator 1.1.1 (core)
salesforcedx 47.1.5 (core)
├─ force-language-services 47.5.0 (core)
└─ salesforce-alm 47.4.0 (core)

**OS and version**:
MacOS Catalina (10.15)

**Comment by tamcdio** [comment]
I've got the same error. Looks like the latest sfdx-cli module declared an incorrect dependency on the salesforcedx version.

npm install sfdx-cli --global
npm ERR! code ETARGET
npm ERR! notarget No matching version found for salesforcedx@47.3.4
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget 
npm ERR! notarget It was specified as a dependency of 'sfdx-cli'

**Comment by KevinGossentCap** [comment]
Sames here.
Additional information : it feels like only a dependency reference problem.
if we look for "salesforcedx" versions:
npm view salesforcedx versions

[...,
  '46.10.2',
  '46.11.0',
  '46.12.1',
  '46.13.1',
  '46.15.3',
  '46.16.0',
  '46.17.0',
  '47.0.7',
  '47.1.4',
  '47.1.5',
  '47.3.6' ]

**Comment by elizabeth-h** [comment]
I am also getting this error when using circleci to push code updates.  It started happening 10/26/2019 after have success all day on 10/25/2019.

**Comment by amtrack** [comment]
This was addressed in [v7.30.12](https://www.npmjs.com/package/sfdx-cli/v/7.30.12).
